wp-cli/wp-cli-bundle
# Remove unused pre-commit scripts

See https://github.com/wp-cli/wp-cli/issues/5480.

This removes the script which sets up the pre-commit hook and its corresponding script. Unlike in https://github.com/wp-cli/wp-cli/pull/5868 here is nothing to remove from composer.json, which means these scripts are not in use anyway.

If you have this pre-commit script installed, you can remove it by deleting the `.git/hooks/pre-commit` file.

## Steps for testing

1. Clone this repo
2. Run `composer install`
3. Confirm no pre-commit hook has been installed into `.git/hooks/pre-commit`
4. Commit some code that would normally fail coding standards checks
5. Confirm there's no error message and the commit succeeds
